lib/index.js

```diff
@@ -170,6 +171,11 @@ function load(options) {
         logger("Cleaning up elements");
         document.getElementById("human-api").remove();
         document.getElementById("human-api-connect-modal-overlay").remove();
+        document.getElementById("app").setAttribute('aria-hidden', 'false');
+        const footer = document.getElementById("footer");
```

Comment:
so here we assume the consumer of connect has an element called `app` and perhaps a `footer` one right? 
I think that applies to our way of using connect in our other apps but might not apply on other client pages

Reply:
As discussed, I will set `aria-hidden` for the element right after body, taking into account that not all apps might have this `app` id